### ISSUE TITLE
function - compute CV

### ISSUE THREAD

**Issue by eburch2** [johnso4/PhishingClassification#9]
compute cross validation given appropriate parameters including the split (if split=70, then training data is 70%, test set is 30%)

if you wanted to be really neat you could add an option to this function loocv=true and combine the other issue on LOOCV with this one because everything else will be the same

return y_hat (predictions), MSE, RSS, Rsquared
